packages/store/addon/-private/system/store.ts

```diff
@@ -73,6 +77,41 @@ type PendingSaveItem = {
 
 let globalClientIdCounter = 1;
 
+function assertRecordsPassedToHasMany(records) {
```

Comment:
make sure you delete these from InternalModel or from store as necessary, currently have duplicates